src/node_internals.h

```diff
@@ -277,6 +277,17 @@ void AppendExceptionLine(Environment* env,
 
 NO_RETURN void FatalError(const char* location, const char* message);
 
+// Like a `TryCatch` but it crashes the process if it did catch an exception.
```

Comment:
Nit: Perhaps this could rephrased to something like "Like a `TryCatch` but crashes the process if an exception was caught"?

Reply:
I'd also write "exits" instead of "crashes" because the latter suggests segfaults and aborts (in my mind at least.)